.ci/scripts/02-init-site-under-test-clone-existing

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -eo pipefail
+
+#
+# This script prepares the site-under-test by cloning the database from
+# an existing site.
+#
+# Use EITHER this script OR the re-install-new script; do not run both.
+#
+
+# Create a new multidev site to test on
+terminus -n env:wake "$TERMINUS_SITE.dev"
+terminus -n build:env:create "$TERMINUS_SITE.dev" "$TERMINUS_ENV" --yes --clone-content
+
+# Run update-db to ensure that the cloned database is updated for the new code.
+terminus -n wp $TERMINUS_SITE.$TERMINUS_ENV -- core update-db
```

Comment:
@greg-1-anderson I had to swap Drush commands for the wp-cli equivalent but I think we could detect the site framework (WordPress or Drupal) and conditionally call the correct one.

Reply:
`BEHAT_PARAMS` was the other items that had to conditionally change. If that were set based on framework as well, I think these scripts could be maintained in one place and used by both images.

Perhaps they could go in https://github.com/pantheon-systems/docker-build-tools-ci?

Reply:
I prefer to keep these scripts committed directly to the example repositories. This makes it easy for folks to customize them, add extra steps, etc.

If there are operations or series of commands that are common to all frameworks, we could hide them in a Terminus Build Tools command. I'm ambivalent about whether to do this with something like Behat. I think it's probably advantageous to show the config steps in longhand, in case someone needs to alter it.

Reply:
> If there are operations or series of commands that are common to all frameworks, we could hide them in a Terminus Build Tools command.

I'm +1 for converting the really short scripts like `.ci/scripts/01-prepare` and `02-init-site-under-test-reinstall-new` into just calls to build tools scripts. For longer scripts like the the Behat and Backstop runners I think they should be moved to a pantheon-maintained composer dependency.

Reply:
I think that a long script could be moved to a Pantheon-maintained subproject, and called from the scripts committed in the example repository.

For any given stage, I'd prefer that all three CI scripts call the same script in the example repository, rather than have the CI scripts call a script in an external Pantheon-maintained repository.

Reply:
In the case of this script, we could maybe make the update-db function a responsibility of build:env:create. It would only have to do it if the multidev was re-used (not created this run) and the `--clone-content` option was specified to copy the database.

For now, I think we should leave this as-is.

Reply:
Card `BT-41` created.